### ISSUE TITLE
[bug] Package inspection doesn't work with python_requires feature in 2.x

### ISSUE THREAD

**Issue by apyvovar** [conan-io/conan#13891]
### Environment details

* Operating System+version: macOS 13.3.1
* Compiler+version: Apple clang version 14.0.3
* Conan version: 2.0.4
* Python version: 3.11.2

### Steps to reproduce

1. Assume, we have the following `conanfile.py`:
```
class MyPackage(ConanFile):
    name = "package"
    version = "0.0.1"
    python_requires = "conan_utils/x.x.x"
    python_requires_extend = "conan_utils.MyConanFileTemplate"
```

2. Note: the package doesn't have attributes `settings`, `options`, `description` etc. (default attributes are defined in `conan_utils`).

3. Calling `conan inspect`  on such package will result in the following output:
```
    "description": null,
    # ...
    "options": null,
    # ...
    "python_requires": "conan_utils/x.x.x",
    "python_requires_extend": "conan_utils.MyConanFileTemplate",
    # ...
    "settings": null,
```

I'm aware, that documentation states, that `conan inspect ... lists class attributes only`, but still, IMO, current issue is a bug, because inspection doesn't represent all package attributes (as in conan 1.x).

**Comment by memsharded** [comment]
Hi @apyvovar 

Thanks for the report.

I think this has been already improved in https://github.com/conan-io/conan/pull/13716, to be released in next 2.0.5 (next week).
If you want to test it from the ``release/2.0`` branch that would be great feedback.

**Comment by memsharded** [comment]
Closing this ticket as solved in 2.0.5 by https://github.com/conan-io/conan/pull/13716, but please re-open or create a new ticker for any further question. Thanks!
